diff --git a/configure.ac b/configure.ac
index e30125b2..63de965b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf 2.71 or later to produce
 dnl a configure script.
-AC_PREREQ([2.71])
+AC_PREREQ([2.69])
 AC_INIT
 AC_CONFIG_SRCDIR([config/config.h.in])
 AC_CONFIG_HEADERS([config/config.h])
@@ -142,9 +142,6 @@ AC_C_CONST
 AC_C_INLINE
 AC_C_RESTRICT
 
-dnl Checks for header files.
-AC_CHECK_INCLUDES_DEFAULT
-
 dnl Check for variadic macro support
 PA_VARIADIC_MACROS
 
